application_form/api/views.py

```diff
@@ -4,10 +4,11 @@
 from rest_framework_simplejwt.authentication import JWTAuthentication
 
 from apartment.elastic.queries import get_apartment_uuids
+from application_form.api.serializers import ApartmentReservationSerializer
 from application_form.api.serializers import (
-    ApartmentReservationSerializer,
-    ApplicationSerializer,
+    ApplicantSerializerBase as ApplicantSerializer,
```

Comment:
For avoiding cognitive load I'd just import `ApplicantSerializerBase` instead of importing `as ApplicantSerializer`.

It will be harder for the next person reading the code to keep track of what `ApplicantSerializer` refers to.